client/index.html

```diff
@@ -98,6 +99,53 @@ <h1 class="title">Sign in to The Lounge</h1>
 								<button type="submit" class="btn">
 									Sign in
 								</button>
+								<% if (signUp) { %>
+								<a href data-target="#sign-up">
+									Sign up
+								</a>
+								<% } %>
+							</div>
+						</div>
+					</form>
+				</div>
+				<div id="sign-up" class="window">
+					<form class="container" method="post" action="">
+						<div class="row">
+							<div class="col-xs-12">
+								<h1 class="title">Sign up to The Lounge</h1>
+							</div>
+							<div class="col-xs-12">
+								<label>
+								Email
+								<input class="input" type="email" name="email" required="required">
+								</label>
+							</div>
+							<div class="col-xs-12">
+								<label>
+								Username
+								<input class="input" name="user" required="required">
+								</label>
+							</div>
+							<div class="col-xs-12">
+								<label class="port">
+								Password
+								<input class="input" type="password" name="password" required="required">
+								</label>
+							</div>
+							<div class="col-xs-12 error" style="display: none;">
+								Sign up failed. May be the username is already used or is invalid.
+								Please retry or contact an administrator.
+							</div>
+							<div class="col-xs-12 success" style="display: none;">
+								Successfully registred. You can <a href data-target="#sign-in">Sign in</a>.
+							</div>
+							<div class="col-xs-12">
+								<button type="submit" class="btn">
+									Sign up
+								</button>
+								<a href data-target="#sign-in">
```

Comment:
TL;DR: I'm not agree with you,  but I'll do what you say.

---
it's not about consistency, it's about UX. 

A link is not a button and conversely.
I mean it's about [affordance](https://en.wikipedia.org/wiki/Affordance) a button should be used to send an event (submit a form for example) and link to anchor to an other place on the app. If I change the link to a button, we will also have a problem with proximity and similarity laws (cf. [gelstalt theory](https://en.wikipedia.org/wiki/Gestalt_psychology#Pr.C3.A4gnanz).